src/index.ts

```diff
@@ -17,12 +18,13 @@ export type ParamMap = Record<string, any>;
 export default function urlcat(baseTemplate: string, params: ParamMap): string;
 
 /**
- * Concatenates the base URL and the path specified using '/' as a separator such
- * that if a '/' occurs at the concatenation boundary in either parameter, it is removed.
+ * Concatenates the base URL and the path specified using '/' as a separator.
+ * If a '/' occurs at the concatenation boundary in either parameter, it is removed.
+ *
+ * @param {String} baseUrl Base URL
+ * @param {String} path Path of URL or query :params
```

Comment:
I wrote "first part" and "second part" here because I wanted to emphasize that you can break up the URL anywhere. The updated description of the `path` parameter isn't correct, query params aren't handled by this overload.